cli/src/test/all-tests.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#set -eu
+set -o pipefail
+
+source "$(dirname "${0}")"/functions-test.sh
+
+MATRIX_OS="$1"
+# Switch IDEasy binary file name based on github workflow matrix.os name (first argument of all-tests.sh)
+BINARY_FILE_NAME="ideasy"
+if [ "${MATRIX_OS}" == "windows-latest" ]; then
+  BINARY_FILE_NAME="ideasy.exe"
+fi
+
+START_TIME=$(date '+%Y-%m-%d_%H-%M-%S')
+
+DEBUG_INTEGRATION_TEST_PREFIX="${HOME}/tmp/ideasy-integration-test-debug"
+DEBUG_INTEGRATION_TEST="${DEBUG_INTEGRATION_TEST_PREFIX}-${START_TIME}"
+IDE_HOME="${DEBUG_INTEGRATION_TEST}/home-dir"
+export IDE_ROOT="${IDE_HOME}/projects"
+IDEASY_DIR="${IDE_ROOT}/_ide"
+FUNCTIONS="${IDEASY_DIR}/installation/functions"
+IDE="${DEBUG_INTEGRATION_TEST}/home-dir/projects/_ide/bin/${BINARY_FILE_NAME}"
+TEST_RESULTS_FILE="${IDE_ROOT}/testResults"
+
+test_files_directory=$(realpath "$0" | xargs dirname)
+
+success=0
+failure=0
+total=0
+
+function doTestsInner() {
+  # Note: requires var test_files_directory to be set.
+  echo "Entering doTestsInner..."
```

Comment:
looks like trace logging. Can we remove this now?
```suggestion
```

Reply:
Removed.